swift/ql/src/queries/Security/CWE-916/InsufficientHashIterations.swift

```diff
@@ -0,0 +1,14 @@
+
+func encrypt() {
+	// ...
+
+	// BAD: Using insufficient (i.e., < 120,000) hash iterations keys for encryption
```

Comment:
We write out "i.e." in full English for clarity:

```suggestion
	// BAD: Using insufficient (that is, < 120,000) hash iterations keys for encryption
```

Reply:
fixed as suggested